pkg/lifecycle/service.go

```diff
@@ -12,7 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package pipeline
+// Package lifecycle wires up everything under the hood of a Conduit instance
+// including metrics, telemetry, logging, and server construction.
+// It should only ever interact with the Orchestrator, never individual
+// services. All of that responsibility should be left to the Orchestrator.
+package lifecycle
```

Comment:
The comment isn't quite accurate, it's probably been taken from somewhere else.

Reply:
@hariso Good catch